src/contracts/jobs/LiquidationJob.sol

```diff
@@ -66,4 +69,11 @@ contract LiquidationJob is Job, Authorizable, Modifiable, ILiquidationJob {
     else if (_param == 'rewardAmount') rewardAmount = _data.toUint256();
     else revert UnrecognizedParam();
   }
+
+  /// @inheritdoc Modifiable
+  function _validateParameters() internal view override {
+    address(liquidationEngine).assertHasCode();
+    address(stabilityFeeTreasury).assertHasCode();
+    rewardAmount.assertNonNull();
```

Comment:
Should `Job` be `Modifiable`, so that `stabilityFeeTreasury` and `rewardAmount` variables can be set and validated in its scope? If so, `_modifyParameters()` and `_validateParameters()` will require a second override.

Reply:
Check out https://github.com/defi-wonderland/hai-core/pull/19/commits/f7343a3b4336e1d4d6e11fcfaf77c44d6bd35791.